tensorrt_llm/_torch/auto_deploy/transformations/transform.py

```diff
@@ -60,6 +56,12 @@ def __call__(self, cm: CachedSequenceInterface) -> nn.Module:
         ############################################################################################
         # RUN MODULAR INFERENCE OPTIMIZER FOR ALREADY-MIGRATED TRANSFORMS
         ############################################################################################
+        # TODO (hg): default values that are not representable in YAML.
+        if "match_attention_layout" in self.ad_config.transforms:
+            self.ad_config.transforms[
+                "match_attention_layout"
+            ].attention_op = AttentionRegistry.get(self.ad_config.attn_backend)
```

Comment:
I was worried we were going to eventually hit a scenario like this where a config value is shared across multiple transforms... 

Let's leave it like this for now to avoid blowing up the PR. But feel free to propose a few ideas about what we can do in order to "share" config values across multiple transforms and we can discuss on slack / during our 1-on-1